colab/tflite_text_classification.ipynb

```diff
@@ -0,0 +1,514 @@
+        "negative_text = \"What a waste of my time.\"\n",
+        "\n",
+        "print(positive_text)\n",
+        "print(tokenize_input(positive_text))"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "This is the best movie I've seen in recent years. Strongly recommend it!\n",
+            "[[   1   13    8    3  117   19  206  109   10 1134  152 2301  385   11\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0    0    0    0    0    0    0    0    0    0    0\n",
+            "     0    0    0    0]]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fvxhY1W7X7Gf"
+      },
+      "source": [
+        "## TFLite\n",
+        "\n",
+        "Overview:\n",
+        "\n",
+        "1.  Load the TFLite model in a [TFLite Interpreter](https://www.tensorflow.org/api_docs/python/tf/lite/Interpreter)\n",
+        "2.   Allocate tensors and get the input and output shape information\n",
+        "3.   Invoke the TFLite Interpreter to test the text classification function"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Nv8x81S5eV5g"
+      },
+      "source": [
+        "interpreter = tflite.Interpreter(\n",
+        "      model_path=str(ARTIFACTS_DIR.joinpath(\"text_classification.tflite\")), \n",
+        "      num_threads=None)\n",
```

Comment:
Can you omit the `num_threads=None`?
These samples don't include it: https://www.tensorflow.org/lite/guide/python#run_an_inference_using_tflite_runtime